core-services/prow/02_config/operator-framework/operator-controller/_prowconfig.yaml

```diff
@@ -0,0 +1,36 @@
+branch-protection:
+  orgs:
+    operator-framework:
+      repos:
+        operator-controller:
+          branches:
+            main:
+              protect: true
+              required_status_checks:
+                contexts:
+                - e2e-kind
+                - unit-test-basic
+                - goreleaser
+                - lint
+                - verify-e2e
+                - extension-developer-e2e
+                - upgrade-e2e
+                - go-apidiff
+tide:
+  merge_method:
+    operator-framework/operator-controller: squash
```

Comment:
Is this the merge method that GH merge queue uses? I figure we should stick with whatever our status quo is, especially since there may be implications for our downstream syncer.

Reply:
Yes, this is what operator-controller uses, so I made sure to maintain it explicitly here.